src/tests/JIT/Methodical/AsgOp/i4/i4.cs

```diff
@@ -670,7 +670,8 @@ private static int f109(int x, int y)
 
 
     [Fact]
-    public static int TestEntryPoint()
+        [OuterLoop]
```

Comment:
The indentation got a bit messed up here

Reply:
Yeah some of these tests had really weird indentation to start with. I'll see if I can normalize it.